core/src/test/scala/io/projectglow/sql/GlowBaseTest.scala

```diff
@@ -43,17 +43,20 @@ abstract class GlowBaseTest
       .set("spark.kryo.registrationRequired", "false")
       .set(
         "spark.hadoop.io.compression.codecs",
-        "org.seqdoop.hadoop_bam.util.BGZFCodec,org.seqdoop.hadoop_bam.util.BGZFEnhancedGzipCodec"
+        classOf[BGZFCodec].getCanonicalName
```

Comment:
Do we not need `BGZFEnhancedGzipCodec` here? I think we definitely have `.gz` files.

Reply:
It was actually an error to include it there. We explicitly add the "enhanced" codec when reading VCF files. We also do not enable the enhanced codecs for all datasources in the genomics runtime: https://docs.databricks.com/release-notes/runtime/3.2.html#changes-and-improvements.